### PR TITLE
[issue#24] Add app.sqlite to .gitignore

### LINKED ISSUE #24: Create DB
**Please describe your feature.**
We need a database to work with.

**Describe the solution you'd like**
    - update .gitignore to ignore the db file
    - Update Docs
    - from file import db
    - db.create_all()

### PR DESCRIPTION
**Pull request by rhyusarmiento**
added app.sqlite to .gitignore
flxes #24

### CODE REVIEW

**Review by Benjibuns — CHANGES_REQUESTED** [review]
thats great for the .gitignore, but please provide documentation and 
- Update README.md in app folder
- from file import db
- db.create_all()

**Comment by rhyusarmiento** [comment]
ok got that done

File: app/README.md [hunk 1]
```diff
@@ -9,4 +9,12 @@ $ pipenv install
...
\ No newline at end of file
+```
+
+## Create DataBase
+- If you need to setup your database you will need to do the following inside a python repl while in your pipenv shell.
+- Make sure to be within the `app` folder directory
+```
+>>> from fileName import db
+>>> db.create_all()
+```
```
Comment (Benjibuns) [inline app/README.md:20]: Add reminder that a new `app.sqlite` file will be created afterwards.

**Review by Benjibuns — CHANGES_REQUESTED** [review]

File: app/README.md (hunk 1 above)
Comment (rhyusarmiento) [reply app/README.md:20]: ok ill work on that

**Review by Benjibuns — APPROVED** [review]
perfect! thank you!

**Pull request merged**